src/Storage/Collection/Taxonomy.php

```diff
@@ -241,7 +245,10 @@ public function getSortorder($field, $slug)
         }
     }
 
-    public function serialize()
+    /**
+     * {@inheritdoc}
+     */
+    public function toArray()
     {
```

Comment:
Ugh, I want to argue you down on this one … and I'm not 100% convinced of the LSP break. However, that's a moot point in my mind as your follow-up question/point makes the most sense … 

So done!